version.php

```diff
@@ -23,7 +23,7 @@
  */
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version  = 2019052801;
+$plugin->version  = 2019061703;
```

Comment:
this doesn't match the savepoint in the upgrade.php script - looks like you might need to fix that :-)